tests/src/test/scala/com/nvidia/spark/rapids/StringFunctionSuite.scala

```diff
@@ -31,15 +31,17 @@ import org.apache.spark.sql.functions._
  * java 11 :  unicode 10.0
  * java 12 :  unicode 11.0    (unsupported)
  * java 13 :  unicode 12.1
+ * java 17 :  unicode 13.0    (unsupported)
  *
  */
 object SupportedUnicodeVersion extends Enumeration {
   val UNICODE_6 = 0
   val UNICODE_10 = 1
-  val UNICODE_12 = 2  
-  
-  val NUM_UNICODE_VERSIONS = 3  
-  val UNICODE_UNSUPPORTED = 4
+  val UNICODE_12 = 2
+  val UNICODE_13 = 3
+
+  val NUM_UNICODE_VERSIONS = 4
+  val UNICODE_UNSUPPORTED = 5
 
   val currentSupportedVersion = UNICODE_12
 }
```

Comment:
Interestingly this class does not use anything from the base class Enumeration. Should we just drop "extends Enumeration" or start using it?

```scala
object SupportedUnicodeVersion extends Enumeration {
  type SupportedUnicodeVersion = Value
  val UNICODE_6,
      UNICODE_10,
      UNICODE_12,
      NUM_UNICODE_VERSIONS,
      UNICODE_UNSUPPORTED = Value

  def currentSupportedVersion = UNICODE_12
}
```

Reply:
Since it's not related to the issue and it's test code, let's refine it later.